ionc/inc/ion_types.h

```diff
@@ -109,6 +109,29 @@ typedef struct ion_type    *ION_TYPE;
 #define tid_SEXP_INT       ION_TYPE_INT(tid_SEXP)
 #define tid_DATAGRAM_INT   ION_TYPE_INT(tid_DATAGRAM)
```

Comment:
Would inverting the definition work for this instead (without adding the `_TYPE` suffix macros?

E.g.
```c
#define tid_NULL_INT   0x000
...
#define tid_NULL  ((ION_TYPE) tid_NULL_INT)
```

This has the benefit of not introducing a separate set of type integers, but I could be missing something more to this change.

Reply:
Yes, that should work.